pkg/apis/serving/v1alpha1/kfservice_framework_tensorrt_test.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2019 kubeflow.org.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+)
+
+var requestedResource = v1.ResourceRequirements{
+	Limits: v1.ResourceList{
+		"cpu": resource.Quantity{
+			Format: "100",
+		},
+	},
+	Requests: v1.ResourceList{
+		"cpu": resource.Quantity{
+			Format: "90",
+		},
+	},
+}
+
+var spec = TensorRTSpec{
+	ModelURI:       "gs://someUri",
+	Resources:      requestedResource,
+	RuntimeVersion: "19.05-py3",
+}
+
+var config = FrameworksConfig{
+	TensorRT: FrameworkConfig{
+		ContainerImage: "someOtherImage",
+	},
+}
+
+func TestCreateModelServingContainer(t *testing.T) {
```

Comment:
I'd expect these unit tests to use golang's best practice table testing(https://github.com/golang/go/wiki/TableDrivenTests), there are a few places not doing this way I will put a separate PR to fix them.

Reply:
Thanks :-)